app/js/actions/labOrdersAction.js

```diff
@@ -31,3 +31,8 @@ export const setOrderLabEncounter = (count, order) => ({
   type: `${SET_ORDER_LAB_ENCOUNTER}_${count}`,
   order,
 });
+
+export const updateOrder = order => ({
+  type: "UPDATE_ORDER",
+  payload: axiosInstance.post(`encounter`, order),
```

Comment:
the post request is better done async

Reply:
@paulvazbad thanks, but it's not so much like a normal ajax post request, else you're right we use redux-promise-middleware, so all the async nature is hidden within it...
feel free to check it out [redux-promise-middleware](https://www.npmjs.com/package/redux-promise-middleware)